docs/how-to/ovn-underlay.rst

```diff
@@ -0,0 +1,64 @@
+================================
+Creating an OVN underlay network
+================================
+
+It is possible to create an OVN underlay network using MicroOVN. The underlay network is the physical network infrastructure that provides connectivity between the nodes in an OVN deployment.
+This underlay networks relies on the OVN Geneve (`Generic Network Virtualization Encapsulation`) tunnel, and is responsible for carrying encapsulated traffic between OVN components.
+This allows the virtual network traffic to be transported over the physical underlay network.
+
+Setting up the underlay network
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To tell MicroOVN to use the underlay network, you need to provide the IP address of the underlay network interface on each node.
+Let us assume that we want to create a three-node OVN cluster and that each node has a dedicated interface ``eth1`` with an IP address.Let says that ``10.0.1.<2,3,4>`` are the respective addresses on the ``eth1`` interface on each node.
```

Comment:
nit: this is just a personal suggestion, but I would suggest using brace expansion for enumerating the IPs, ``10.0.1.{2,3,4}``

nit2: space after comma `... address.Let ...`

Reply:
fixed